src/components/popover/Popover.types.ts

```diff
@@ -26,4 +27,5 @@ export interface PopoverContentProps {
   children?: ReactNode;
   id?: string;
   root?: HTMLElement | null | MutableRefObject<HTMLElement | null>;
+  border?: BorderType | null;
```

Comment:
I'd set the type as:
```suggestion
  border?: BorderType;
```
Is there any reason to support `null` value?